demos/python_demos/multi_camera_multi_person_tracking/multi_camera_multi_person_tracking.py

```diff
@@ -17,18 +17,56 @@
 from threading import Thread
 import json
 import logging as log
+import os
+import random
 import sys
 
 import cv2 as cv
 
-from utils.network_wrappers import Detector, VectorCNN
+from utils.network_wrappers import Detector, VectorCNN, MaskRCNN, DetectionsFromFileReader
 from mc_tracker.mct import MultiCameraTracker
-from utils.misc import read_py_config
-from utils.video import MulticamCapture
-from utils.visualization import visualize_multicam_detections
-from openvino.inference_engine import IECore # pylint: disable=import-error,E0611
+from utils.analyzer import save_embeddings
+from utils.misc import read_py_config, check_pressed_keys, AverageEstimator, set_log_config
+from utils.video import MulticamCapture, NormalizerCLAHE
+from utils.visualization import visualize_multicam_detections, get_target_size
+from openvino.inference_engine import IECore  # pylint: disable=import-error,E0611
 
-log.basicConfig(stream=sys.stdout, level=log.DEBUG)
+set_log_config()
+
+
+def check_detectors(args):
+    detectors = {
+        '--m_detector': args.m_detector,
+        '--m_segmentation': args.m_segmentation,
+        '--detections': args.detections
+    }
+    non_empty_detectors = [(det, value) for det, value in detectors.items() if value]
+    det_number = len(non_empty_detectors)
+    if det_number == 0:
+        log.error('No detector specified, please specify one of the following parameters: '
+                  '\'--m_detector\', \'--m_segmentation\' or \'--detections\'')
+    elif det_number > 1:
+        det_string = ''.join('\n\t{} = {}'.format(det[0], det[1]) for det in non_empty_detectors)
```

Comment:
```suggestion
        det_string = ''.join('\n\t{}={}'.format(det[0], det[1]) for det in non_empty_detectors)
```

You don't use spaces when specifying the parameters, so they should also not be used when printing them.

Reply:
Fixed.